### ISSUE TITLE
Initialization

### ISSUE THREAD

**Issue by GOATmessi8** [unsky/focal-loss#4]
What is your initialization of the detector? Is that exactly the same with original paper to set bias = -log((1-pi)/pi) or use normal softmax for several times like your previous claimed?

**Comment by GOATmessi8** [comment]
btw, it seems you mul the same alpha to all examples, but to my understand, alpha should be different for pos and neg examples

**Comment by unsky** [comment]
i explore the method in proposal + classification, not the regression(like ssd), the initialization is a method for early training to balanced the heavy inbalanced examples, but my goal is to compare the method  with ohem in hards mining, the initialization is not used in my experiment, and the effect of alpha is equal to the effect of learning rate. if you have other idea about alpha, can you tell me how to?
my experiment is:does focal loss doing well in proposal +classification framework?

**Comment by GOATmessi8** [comment]
@unsky Thanks! I have done some experiment on regression framework, and I just mul (1-alpha) to pos examples, causing a tiny improvement though.

**Comment by gunshi** [comment]
@ruinmessi if you don't mind sharing, what were your conclusions on the comparison(OHEM vs focal loss, and how focal loss does on proposal+classification)? I'm curious about something similar..
